IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -38,6 +39,10 @@
 #import "MSIDKeychainUtil.h"
 #import "MSIDDefaultAccountCacheKey.h"
 #import "MSIDAppMetadataItemSerializer.h"
+#import "MSIDMacSharedCredentialCacheItem.h"
+#import "MSIDMacAppCredentialCacheItem.h"
+#import "MSIDMacAppCredentialItemSerializer.h"
+#import "MSIDCacheItemJsonSerializer.h"
 #import "MSIDAccountMetadataCacheKey.h"
 
 /**
```

Comment:
The `MSIDMacKeychainTokenCache` comment block needs to be updated to match the revised code.

Reply:
In particular, the "Type 3" description is out of date since they're no longer stored as individual keychain items.

Reply:
Updated.